README.md

```diff
@@ -1,3 +1,11 @@
 # mediawiki-agent
 
 [Smolagents](https://smolagents.org) components to interact with [pywikibot](https://pypi.org/project/pywikibot/).
+
+## Installation
+
+To install the necessary dependencies, run the following command:
+
+```bash
+pip install smolagents pywikibot
```

Comment:
Since the pyproject.toml specifies the dependencies, this seems less than ideal.